model/ServerHandler.php

```diff
@@ -0,0 +1,167 @@
+<?php
+# $Id: DomainHandler.php 2016-03-11 00:16:00Z jan-frederik_rieckers $
+
+/**
+ *
+ */
+class ServerHandler extends PFAHandler{
+
+    protected $db_table = 'server';
+    protected $id_field = 'server';
+
+    protected function validate_new_id() {
+        return true;
+    }
+
+    protected function no_domain_field() {
+        # PFAHandler die()s if domain field is not set. Disable this behaviour for AdminHandler.
+    }
+
+    protected function initStruct() {
+        $super = $this->is_superadmin;
+
+        $this->struct=array(
+            # field name            allow       display in...   type    $PALANG label  $PALANG description
+            #                       editing?    form    list
+            'server'      => pacol( $this->new, 1,      1,      'text', 'server',        ''),
+            'description' => pacol( $super,     1,      1,      'text', 'description',   ''),
+            'address'     => pacol( $super,     $super, $super, 'text', 'smtpaddress',   'pAdminEdit_server_address_text'),
+            'created'     => pacol( 0,          0,      0,      'ts',   'created',       ''),
+            'modified'    => pacol( 0,          0,      $super, 'ts',   'last_modified', ''),
+            '_can_edit'   => pacol( 0,          0,      1,      'int',  '',              '', 0,
+              /*options*/ '',
+              /*not_in_db*/ 0,
+              /*dont_write_to_db*/ 1,
+              /*select*/ $this->is_superadmin . ' as _can_edit' ),
+            '_can_delete' => pacol( 0,          0,      1,      'int',  '',              '', 0,
+              /*options*/ '',
+              /*not_in_db*/ 0,
+              /*dont_write_to_db*/ 1,
+              /*select*/ $this->is_superadmin . ' as _can_delete' ),
+        );
+    }
+
+    protected function initMsg() {
+        $this->msg['error_already_exists'] = 'pAdminCreate_server_server_text_error';
```

Comment:
Feel free to use something like 'server_already_exists' as message name - the pAdmin* message names are only there for historical reasons[tm]. I prefer names that relate more to the message content, and the only reason why the pAdmin* messages still have this name is that nobody renamed them.

Anyway, there's no reason to use hard to understand pAdmin* message names for new messages.

This comment also applies to the other pAdmin* messages you added in this function and in webformConfig().